Userland/DevTools/UserspaceEmulator/Emulator_syscalls.cpp

```diff
@@ -1533,6 +1533,7 @@ u32 Emulator::virt$allocate_tls(FlatPtr initial_data, size_t size)
     // TODO: This matches what Thread::make_thread_specific_region does. The kernel
     // ends up allocating one more page. Figure out if this is intentional.
     auto region_size = align_up_to(size, PAGE_SIZE) + PAGE_SIZE;
+    m_range_allocator.reserve_user_range(VirtualAddress(0x20000000), region_size);
```

Comment:
same here? (Or maybe even add it to `Kernel/API/MemoryLayout.h`?)

Reply:
This constant seems completely arbitrary, and isn't used in the kernel. In the kernel, the thread-specific region gets put at the lowest available memory address, which is different across runs. I don' think adding ASLR to UE would have any benefits though.

+1 for the `constexpr` variable; will fix.